apps/tools/.env.local

```diff
@@ -0,0 +1,6 @@
+NODE_PROVIDER_TYPE=http
+NODE_URL= https://alfajores-forno.celo-testnet.org
+
+WALLET_ADDRESS=0x180bc870A789932eF5Fcc28C4E5B509Be200d1e2
+WALLET_PRIVATE_KEY=0x3687c831ccfc4d8665ad5283e3e9d8486aa23fcdc7b2e383323ee8a110080436
```

Comment:
Probably fine to check-in as long as we're not using for actual alfajores environment and just for dev testing

Reply:
Even so should be in .env.local.example just to keep things clean.